assets/js/carousel.js

```diff
@@ -40,9 +40,15 @@
             .css("pointer-events", "auto");
 
         $(".carousel-item-label")
-            .removeClass("bg-purple-700")
+            .removeClass("border-purple-700")
```

Comment:
Still need to do some refactoring for better maintainability here and in a few other places. Will do that post-merge unless objections.